pkg/utils/time_test.go

```diff
@@ -0,0 +1,79 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestParseDuration(t *testing.T) {
+	tests := []struct {
+		name    string
+		s       string
+		want    time.Duration
+		wantErr bool
+	}{
+		{
+			name:    "duration string's time unit is 'd'",
+			s:       "1d",
+			want:    time.Hour * 24 * time.Duration(1),
+			wantErr: false,
+		},
+		{
+			name:    "duration string's time unit is not 'd'",
+			s:       "300ms",
+			want:    300 * time.Millisecond,
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Would you please add a sample for a error case.

Reply:
done